src/i965_drv_video.h

```diff
@@ -81,6 +81,7 @@
 #define ENCODER_HIGH_QUALITY      ENCODER_DEFAULT_QUALITY
 #define ENCODER_LOW_QUALITY       2
 
+#define I965_MIN_CODEC_ENC_RESOLUTION_WIDTH          32
```

Comment:
The name of this macro does not convey its usage.  That is, it's also used to constrain minimum *height* in your code.

Reply:
thanks